client/jetpack-connect/site-topic.js

```diff
@@ -26,7 +28,15 @@ class JetpackSiteTopic extends Component {
 
 		this.props.saveSiteVertical( siteId, siteVertical );
 
-		page( `/jetpack/connect/plans/${ siteSlug }` );
+		const jpVersion = this.props.siteJetpackVersion;
+		if ( ! jpVersion ) {
+			return null;
```

Comment:
So if we don't have the version, we'll just bail? Aso, in which condition would that happen? Can we even show this form if we don't have a site loaded?